packages/core/src/index.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {AndroidSdkTools} from './lib/androidSdk/AndroidSdkTools';
+import {Config} from './lib/Config';
+import {GradleWrapper} from './lib/GradleWrapper';
+import Log from './lib/Log';
```

Comment:
Why no {}?

Reply:
`Log` is using `export default Log`, while others simple use `export`. Maybe I should change `Log` to just use `export`, but likely in another PR.